src/analyses/unassumeAnalysis.ml

```diff
@@ -23,7 +23,6 @@ struct
   let exitstate _ = D.empty ()
 
   let context _ _ = ()
-  let should_join _ _ = false
 
```

Comment:
Do we not need some equivalent version of this?

Reply:
I couldn't figure out why it even was `false` to begin with. I don't recall ever using the unassume analysis in path sensitive mode. Also a constant `false` doesn't make sense I think because it isn't even reflexive.